src/views/authenticated/List.tsx

```diff
@@ -53,26 +54,26 @@ export function List({ data: unfilteredListItems, listPath }: Props) {
 				Hello from the <code>/list</code> page!
 			</p>
 
-			<div>
-				<section>
-					{unfilteredListItems.length > 0 && (
-						<FilterListInput
-							searchTerm={searchTerm}
-							setSearchTerm={setSearchTerm}
-						/>
-					)}
-					<h3>Want to add more items to your list?</h3>
-					<button
-						onClick={() => navigate("/manage-list")}
-						aria-label="Navigate to add more items to your list"
-					>
-						{"Add items"}
-					</button>
-				</section>
+			<section className="sticky-top bg-dark">
```

Comment:
This is not the right background color but needed to set it to something to see the scrolling and make sure ti stuck to the top correctly!